src/ray/rpc/gcs_server/gcs_rpc_client.h

```diff
@@ -87,11 +87,12 @@ class Executor {
 /// The priority of timeout is each call > handler > whole service
 /// (the lower priority timeout is overwritten by the higher priority timeout).
 /// \param SPECS The cpp method spec. For example, override.
+/// \param IS_INSECURE Whether to attach a cluster_id token to the metadata of the call.
 ///
 /// Currently, SyncMETHOD will copy the reply additionally.
 /// TODO(sang): Fix it.
-#define VOID_GCS_RPC_CLIENT_METHOD(                                                      \
-    SERVICE, METHOD, grpc_client, method_timeout_ms, SPECS)                              \
+#define _VOID_GCS_RPC_CLIENT_METHOD(                                                     \
+    SERVICE, METHOD, grpc_client, method_timeout_ms, SPECS, IS_INSECURE)                 \
```

Comment:
Actually why do we need to change client side?

Reply:
removed, can use Nil